api/managementpb/actions.proto

```diff
@@ -20,86 +20,132 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 };
 option go_package = "managementpb";
 
-import "agentpb/agent.proto";
 import "github.com/mwitkow/go-proto-validators/validator.proto";
 import "google/api/annotations.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
-// RunActionRequest describes request to run an action.
-message RunActionRequest {
-  // PMM agent instance identifier.
-  string pmm_agent_id = 1;
-  // Service ID for which action is running.
-  string service_id = 2;
-  // Node ID for which action is running.
-  string node_id = 3;
-  // Action name to run.
-  agent.ActionName action_name = 4;
```

Comment:
In all other public APIs for creating/changing something, we use type in the API name.